### PR TITLE
samples: Bluetooth: hci_usb: Disable USB_CDC_ACM

### PR DESCRIPTION
**Pull request by cvinayak**
BlueZ btusb does not recognize hci_usb sample as Bluetooth Controller when CDC ACM is enabled in the sample.

Refer to https://github.com/zephyrproject-rtos/zephyr/issues/29107

https://github.com/zephyrproject-rtos/zephyr/issues/29107#issuecomment-1048705158 :
> you cannot use hci_usb and CDC ACM in composite configuration with btusb.

### CODE REVIEW

**Review by Thalley — COMMENTED** [review]
Is this a new change? I've used the nRF52840 with Bluez using the `hci_usb` sample for years without any issues

**Review by nordicjm — APPROVED** [review]

**Comment by tmon-nordic** [comment]
Is the issue you are referring to essentially fixed by [Bluetooth: btusb: Do not require hardcoded interface numbers](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/bluetooth/btusb.c?id=c13380a55522bf14e54779a142487c224509db95) that will be included in Linux 6.4? Or is there any other reason?

**Comment by cvinayak** [comment]
@Thalley 

> Is this a new change? I've used the nRF52840 with Bluez using the `hci_usb` sample for years without any issues

I believe the regression could have been introduced by 897d2b82. I did not bisect (only based on the lines I had to update in this PR)


@tmon-nordic 

> Is the issue you are referring to essentially fixed by [Bluetooth: btusb: Do not require hardcoded interface numbers](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/drivers/bluetooth/btusb.c?id=c13380a55522bf14e54779a142487c224509db95) that will be included in Linux 6.4? Or is there any other reason?

I hope will be fixed by the changes in btusb. But in general `hci_usb` sample is intended to be a HCI controller-only, and if need be (when btusb fix is in the open) users can explicitly enable the console when required (say for debug logging).

**Review by Thalley — APPROVED** [review]

**Review by tmon-nordic — APPROVED** [review]

File: samples/bluetooth/hci_usb/prj.conf
Comment (jfischer-no) [inline samples/bluetooth/hci_usb/prj.conf:6]: okay, it does not fix a bug and is a problematic change, merged without maintainer approval, this patch must be reverted.

Comment (cvinayak) [reply samples/bluetooth/hci_usb/prj.conf:6]: What is the problem with disabling console?

**Pull request merged**
